misc/review

```diff
@@ -126,11 +126,29 @@ inspect_cmd_skip ()
     echo "SKIPPING $1" 1>&2
 }
 
-inspect_cmd_show_diff ()
+inspect_cmd_show_genenric ()
```

Comment:
Haven't been of much help lately. The least I can do is warn you of a misspelling when I spot it!